### ISSUE TITLE
Programatically fill a menu with file contents of a folder

### ISSUE THREAD

**Issue by monkeyswarm** [monkeyswarm/MobMuPlat#70]
Request file contents of the app Document folder, or a subfolder, and return list of filenames in order to populate a menu

**Comment by dfkettle** [comment]
Yes please! That's what I was trying to do with [openpanel], until you told me that [openpanel] doesn't work in libpd. I want to present a list of "presets" or "patches" to the user and allow them to pick one.

**Comment by blindmanonacid** [comment]
I think this should be possible if it is compiled with the latest version of libpd, which should include [file glob] and [pdcontrol] among other things!
